### ISSUE TITLE
Add `theme.defaultFgColor`

### ISSUE THREAD

**Issue by simon-abbott** [jesseduffield/lazygit#2279]
**Is your feature request related to a problem? Please describe.**

#2257 switched to using the default terminal color instead of pre-defined colors for light/dark mode. The issue is that some terminal themes, like mine, don't lend themselves well to this theming.

**Describe the solution you'd like**

Add and use a new `theme.defaultFgColor` setting to allow us to override the default theming.

**Describe alternatives you've considered**

Not having this setting is technically possible, but it doesn't look nearly as good.

**Additional context**

Can't add screenshots, unfortunately, but trust me, it looks bad. If you have a mac try setting your terminal theme to 'homebrew' then use lazygit.

**Comment by jesseduffield** [comment]
> Can't add screenshots, unfortunately, but trust me, it looks bad

😆 

As discussed, the `theme.defaultFgColor` is a good approach. This is a super-simple issue. Here's what needs doing:
1. 
in `pkg/theme/theme.go:61` we want to go:
```go
DefaultTextColor = GetTextStyle(themeConfig.DefaultFgColor, false)
GocuiDefaultTextColor = GetGocuiStyle(themeConfig.DefaultFgColor)
```

2.
Then we just need to actually have `DefaultFgColor` defined on our theme config which you can add in `pkg/config/user_config.go:56`. 

3. 
Then in the same file at line `pkg/config/user_config.go:359` we want to give that field a default of `[]string{"default"}`

4.
Then in `docs/Config.md:31` we need to add that default value there too

I've chucked a good-first-issue on this ticket and if you take a look [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#running-in-a-github-codespace) you'll see how to spin up a codespace to add this feature from within your browser!

Would you be keen to give this a go @simon-abbott ?

**Comment by simon-abbott** [comment]
I'd be happy to, assuming I can find the time in my day to work on it!

**Comment by arnaudperalta** [comment]
Hi @jesseduffield, I would enjoy to work on a first issue on lazygit. Can I work on this one or is this only assigned for OP ?

**Comment by jesseduffield** [comment]
@simon-abbott if you haven't made a start yourself we can assign to @arnaudperalta :)

**Comment by simon-abbott** [comment]
@arnaudperalta Go for it!

**Comment by simon-abbott** [comment]
I can confirm that this is fixed in `v0.37.0`! 🎉
